Controls/Input/Input.UWP/NumericBox/RadNumericBox.cs

```diff
@@ -929,6 +929,8 @@ private void OnTextBoxTextChanged(object sender, TextChangedEventArgs e)
         {
             this.UpdateVisualState(true);
             this.ValidateText();
+            if (this.GetBindingExpression(ValueProperty)?.ParentBinding.UpdateSourceTrigger == UpdateSourceTrigger.PropertyChanged && !String.IsNullOrWhiteSpace(this.TextBox.Text))
```

Comment:
The suggested solution is not an appropriate one as it will cause a small unexpected behavior. For example if you have maximum value set to 100 and a value of 500 is entered the text will be changed to 100 according to the updated value. However, Text should be changed after the focus is lost with that minimum value (not while typing) or a breaking change could be introduced. 

That behavior is caused by the UpdateTextBoxText called inside the OnValueChagned method called when the Value is changed. Now as the value could be updated when the property gets changed we should prevent updating the Text as well.
`  if (oldValue != numericBox.Value)
            {
                numericBox.UpdateTextBoxText();
                numericBox.OnValueChanged();
            }`